pandas/io/json/_normalize.py

```diff
@@ -531,7 +531,14 @@ def _recursive_extract(data, path, seen_meta, level=0):
             raise ValueError(
                 f"Conflicting metadata name {k}, need distinguishing prefix "
             )
-        result[k] = np.array(v, dtype=object).repeat(lengths)
+        if v and isinstance(v[0], abc.Iterable):
+            out = []
+            for item, repeat in zip(v, lengths):
+                for _ in range(repeat):
+                    out.append(item)
+        else:
+            out = np.array(v).repeat(lengths)
```

Comment:
The type checking CI fails. `(expression has type "ndarray[Any, dtype[Any]]", variable has type "List[Any]")` Should I still add `.tolist()` after repeat? @mroeschke

Reply:
Using a `# type: ignore[particular-violation]` here is okay imo